source.mak

```diff
@@ -73,7 +74,7 @@ SOURCES = \
 
 ENVIRONMENT_HEADERS = e_msoft.h
 
-ENVIRONMENT_SOURCES = 
+ENVIRONMENT_SOURCES =
```

Comment:
this change probably shouldn't be included (I imagine your editor stripped trainling spaces, and it's not so bad, but it's a little irrelevant).  but maybe we don't care